tests/Integrations/Guzzle/V6/GuzzleIntegrationTest.php

```diff
@@ -125,8 +141,18 @@ public function testDistributedTracingIsPropagated()
 
         // trace is: custom
         $this->assertSame($traces[0][0]['span_id'], (int) $found['headers']['X-Datadog-Trace-Id']);
+
         // parent is: curl_exec, used under the hood
-        $this->assertSame($traces[0][2]['span_id'], (int) $found['headers']['X-Datadog-Parent-Id']);
+        $curl_exec = null;
```

Comment:
Total nit: 🐫 And same for the v5 test. 😉

Reply:
Obviously this was another case of me forgetting this style piece, but in this case 'curl_exec' is the operation name -- wouldn't it make more sense to match that?

Reply:
Yeah, that's why I approved the PR with this bit being optional. My personal preference to to keep all vars with a consistent style, but given our past conversations about naming, I figured you'd take exception here. Overall I don't have a strong preference either way. Just follow your heart. 😄